qkeras/utils.py

```diff
@@ -406,10 +417,17 @@ def quantize_rnn(layer):
     elif layer["class_name"] in ["SimpleRNN", "LSTM", "GRU"]:
       quantize_rnn(layer)
 
-    elif layer['class_name'] == 'QBidirectional':
-      quantize_rnn(layer['layer'])
-      if "backward_layer" in layer:
-        quantize_rnn(layer['backward_layer'])
+    elif layer['class_name'] == 'Bidirectional':
```

Comment:
do you have python tests for this newly added code?

Reply:
Now we do